setup.py

```diff
@@ -274,6 +274,7 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
 # DEPENDENCIES_EPOCH_NUMBER in the Dockerfile
 ############################################################################################################
 devel = [
+    'astroid~=2.2.5',
```

Comment:
Since this is a dependency we do not directly use in code I think a comment behind it would be helpful to know why it is has been added and even more important when we can remove it. It is because of the current pylint version, right?

Reply:
Good call. It is the pylint indeed.